pathogen detection from (direct Nanopore) sequencing data using Galaxy - Updating the training with FAQ and adding an explanation concept (#3954)
galaxyproject/training-material: topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/faqs/importing_genomes.md

---

> title: How can I import other genomes?

Author: Where not How

box_type: question
layout: faq
contributors: [EngyNasr]
---

> For host filtering, you can find the [full list of possible host sequences](https://github.com/lskatz/Kalamari/blob/master/src/Kalamari_v3.9.1.tsv) that can be removed. In this workshop we used kalamari DB, so reads are either tagged to map one of those species or are left unassigned. If the task at hand in the real world cannot be covered by those, you can also try another DB for kraken2 that includes your species (or maybe retain unmapped reads from a read aligner such as bowtie2, minimap2…).

Reviewer: ```suggestion
In this tutorial, we used kalamari DB with the [full list of possible host sequences](https://github.com/lskatz/Kalamari/blob/master/src/Kalamari_v3.9.1.tsv) that can be removed. Reads are either tagged to map one of those species or are left unassigned. If the task at hand in the real world cannot be covered by those, you can also try another DB for Kraken2 that includes your species (or maybe retain unmapped reads from a read aligner such as Bowtie2, Minimap2…).
```
